content/kapp-controller/docs/latest/packaging-workflows.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Packaging Workflows
+---
+
+We recommend reading through the docs about the new [packaging
+apis](packaging.md) to familiarize yourself with the yaml used in these
+workflows.
+
+## Package Authoring
+
+## Package Consumption
+
+This workflow walks through the example contained in
+the [`packaging-demo`](https://github.com/vmware-tanzu/carvel-kapp-controller/tree/dev-packaging/examples/packaging-demo)
+directory and assumes all commands are run from within the directory.
+
+### Setup
+
+First, some [setup of rbac](security-model.md) is necessary and can be done using:
+
+```
+$ kapp deploy -a default-ns-rbac -f https://raw.githubusercontent.com/vmware-tanzu/carvel-kapp-controller/develop/examples/rbac/default-ns.yml
+```
+
+### Adding the Repo
+
+Now that we are setup, we need to make some packages available to the cluster.
+To do this, we can apply the contents of `repo.yml`, which contains a
+PackageRepository CR:
+
+```
+$ kapp deploy -a repo -f repo.yml
+```
+
+Now we can list the packages and see what's available. It will look something like this:
+
+```
+$ kubectl get packages
+NAME                              PUBLIC-NAME            VERSION      AGE
+pkg2.test.carvel.dev.1.0.0        pkg2.test.carvel.dev   1.0.0        7s
+pkg2.test.carvel.dev.2.0.0        pkg2.test.carvel.dev   2.0.0        7s
+pkg2.test.carvel.dev.3.0.0-rc.1   pkg2.test.carvel.dev   3.0.0-rc.1   7s
```

Comment:
I think these were all renamed to `pkg` instead of `pkg2`.

Reply:
Yes, good catch, I will update those.